diff --git a/chrome/browser/importer/importer_uma.cc b/chrome/browser/importer/importer_uma.cc
index d858676745103908f16a41392c0da42c3683c555..1943125f05f6b3a15cbad236e43311164ba2a7a6 100644
--- a/chrome/browser/importer/importer_uma.cc
+++ b/chrome/browser/importer/importer_uma.cc
@@ -59,6 +59,15 @@ void LogImporterUseToMetrics(const std::string& metric_postfix,
     case TYPE_BOOKMARKS_FILE:
       metrics_type = IMPORTER_METRICS_BOOKMARKS_FILE;
       break;
+    case TYPE_CHROME:
+      // TODO: Wire this up if we want to record metrics on users who import from Chrome
+      break;
+    case TYPE_VIVALDI:
+      // TODO: Wire this up if we want to record metrics on users who import from Chrome
+      break;
+    case TYPE_OPERA:
+      // TODO: Wire this up if we want to record metrics on users who import from Chrome
+      break;
   }
 
   // Note: This leaks memory, which is the expected behavior as the factory
