diff --git a/chrome/browser/flags/android/chrome_feature_list.cc b/chrome/browser/flags/android/chrome_feature_list.cc
index 293ce4a18131c4a7d1f0d8cb614cc981f12f6996..4d1c4ad4e4094b38a8b20419bd2b73a4ed223921 100644
--- a/chrome/browser/flags/android/chrome_feature_list.cc
+++ b/chrome/browser/flags/android/chrome_feature_list.cc
@@ -389,6 +389,7 @@ const base::Feature* const kFeaturesExposedToJava[] = {
 };
 
 const base::Feature* FindFeatureExposedToJava(const std::string& feature_name) {
+  BRAVE_FIND_FEATURE_EXPOSED_TO_JAVA
   static auto kFeaturesExposedToJavaMap = base::NoDestructor(
       base::MakeFlatMap<base::StringPiece, const base::Feature*>(
           kFeaturesExposedToJava, {},
