diff --git a/third_party/blink/public/mojom/permissions_policy/permissions_policy_feature.mojom b/third_party/blink/public/mojom/permissions_policy/permissions_policy_feature.mojom
index 1296a93ff1407fcac503ee18d0fb849f7731ceb8..bfeb7496f3cc2027602a92c21b792fc565299f2c 100644
--- a/third_party/blink/public/mojom/permissions_policy/permissions_policy_feature.mojom
+++ b/third_party/blink/public/mojom/permissions_policy/permissions_policy_feature.mojom
@@ -214,6 +214,8 @@ enum PermissionsPolicyFeature {
   // Controls use of FedCM API.
   kFederatedCredentials = 102,
 
+  kEthereum = 4096, kSolana = 4097,
+
   // Don't change assigned numbers of any item, and don't reuse removed slots.
   // Add new features at the end of the enum.
   // Also, run update_permissions_policy_enum.py in
